### PR TITLE
Improve docstring with params/returns/raises - attempt at #15370

### PR DESCRIPTION
**Pull request by GaurangTandon**
#### References to other Issues or PRs

#15370 

#### Brief description of what is fixed or changed

Added params/returns/raises for docstring of `continuous_domain` in `calculus/util.py`.

#### Other comments

This is my first time cotributing to the sympy repository, I hope I have correctly improved the docstring as was required. If it is correct, please merge it, as after it I intend to make another PR containing few more docstring improvements.

#### Release Notes

* calculus 
  * Improve docstring for `continuous_domain` in `util.py`

### CODE REVIEW

**Comment by GaurangTandon** [comment]
Could anyone please help me understand why my Travis CI build is failing? I have already made two commits attempting to fix the issue of trailing whitespace, but I cannot understand why the third build is failing.

File: sympy/calculus/util.py [hunk 1]
```diff
@@ -37,6 +47,16 @@ def continuous_domain(f, symbol, domain):
...
 
+    Returns
+    =======
+    Intervals where the function is continuous.
+
+    Raises
+    ======
+    NotImplementedError
+        if the method to determine continuity of such a function
+        has not yet been developed
```
Comment (moorepants) [inline sympy/calculus/util.py]: I think it is best to make these sentences starting with capital letters and ending with periods. Make sure all are consistent.

File: sympy/calculus/util.py [hunk 2]
```diff
@@ -37,6 +47,16 @@ def continuous_domain(f, symbol, domain):
     >>> continuous_domain(log(2*x - 1), x, S.Reals)
     Interval.open(1/2, oo)
 
+    Returns
+    =======
+    Intervals where the function is continuous.
```
Comment (moorepants) [inline sympy/calculus/util.py]: Give a type here.

**Comment by GaurangTandon** [comment]
@moorepants please have a look at the latest updates.

File: sympy/calculus/util.py [hunk 3]
```diff
@@ -49,13 +49,15 @@ def continuous_domain(f, symbol, domain):
 
     Returns
     =======
-    Intervals where the function is continuous.
+
+    interval
```
Comment (moorepants) [inline sympy/calculus/util.py]: Is this a sympy type?

Comment (GaurangTandon) [reply sympy/calculus/util.py]: Sorry, my bad. So, now I've looked at [test_util.py](https://github.com/sympy/sympy/blob/master/sympy/calculus/tests/test_util.py) and it seems that the correct types for both the argument "domain" and the return type are `Interval` (I also looked at `sets/sets.py`). I have made another commit. Hope it's correct

**Comment by moorepants** [comment]
Thanks.

**Pull request merged**